### ISSUE TITLE
I removed a field for testing and cant re-add that field to my model, help! 

### ISSUE THREAD

**Issue by leviross** [balderdashy/sails-mongo#263]
Im running Sails/Angular/Mongo and I did a Mongo command from the shell yesterday: db.user.update(
   { id: "123" },
   { $unset: { userCart: [] } }
)

That removed the userCart field from my User model apparently and I can only 'set' it again on existing users. New users will have no field 'userCart' and when you try to add an item to their cart, there will be an error. 

I tried everything from changing schema to 'true' and even deleted my model and re-generated it. 

I have no clue why there is no field 'userCart' on any user now. Please help, I love Sails and I am really stuck on this one:)

**Comment by sailsbot** [comment]
Thanks for posting, @leviross.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
